go.mod

```diff
@@ -4,14 +4,18 @@ go 1.18
 
 require (
 	github.com/go-logr/logr v1.2.3
+	github.com/golang/mock v1.6.0
 	github.com/google/go-cmp v0.5.8
 	github.com/google/nftables v0.0.0-20220422151220-76ed01e300f2
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/ks2211/go-suricata v0.0.0-20200823200910-986ce1470707
 	github.com/metal-stack/metal-go v0.16.6
 	github.com/metal-stack/metal-lib v0.9.0
-	github.com/metal-stack/metal-networker v0.8.3
+	github.com/metal-stack/metal-networker v0.8.4-0.20220530124700-e638aa05ed41
```

Comment:
So we need a metal-networker release too. I can try to review, but I can't guarantee best results. :sweat_smile: